README.md

```diff
@@ -32,7 +32,7 @@ __gtk__ expects __GTK+__, __GLib__ and __Cairo__ development files to be install
 ### Windows
 
 Install [mingw-w64](http://mingw-w64.yaxm.org/) (select the win32 threading model) and download a __GTK+__ SDK:
- * The GNOME project has an official distribution of GTK+ 3.6: [x86](http://www.gtk.org/download/win32.php), [x64](http://www.gtk.org/download/win64.php).
+ * The GNOME project has official distributions of GTK+ 3.4 upto 3.10: http://win32builder.gnome.org/
```

Comment:
I'd prefer if the link was put "inside" "3.4 upto 3.10". Example:

The GNOME project has official distributions of GTK+ [3.4 upto 3.10](http://win32builder.gnome.org/).